satellite/console/projectinvoicestamps.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package console
+
+import (
+	"context"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// ProjectInvoiceStamps exposes methods to manage ProjectInvoiceStamp table in database
+type ProjectInvoiceStamps interface {
+	Create(ctx context.Context, stamp ProjectInvoiceStamp) (*ProjectInvoiceStamp, error)
+	GetByProjectIDStartDate(ctx context.Context, projectID uuid.UUID, startDate time.Time) (*ProjectInvoiceStamp, error)
+	GetAll(ctx context.Context, projectID uuid.UUID) ([]ProjectInvoiceStamp, error)
+}
+
+// ProjectInvoiceStamp declare invoice to project mappings
+type ProjectInvoiceStamp struct {
```

Comment:
What's a stamp?

Reply:
It is like a notice/mark/reference of the created invoice

Reply:
Let's add a comment describing on what it is and what it is being used for.